test/extended/networking/isolation.go

```diff
@@ -7,13 +7,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// This test requires a network plugin that supports namespace isolation.
-// NOTE: if you change the test description, update networking.sh too!
 var _ = Describe("[networking] network isolation plugin", func() {
 	f1 := e2e.NewFramework("net-isolation1")
 	f2 := e2e.NewFramework("net-isolation2")
 
-	It("should prevent communication between pods in different namespaces", func() {
+	var expectation string
+	if isMultitenant() {
```

Comment:
my bad, it was the `e2e.NewFramework()` calls that were the problem, and I can just leave those outside the helper